source/includes/stackpath/_workloads.md

```diff
@@ -220,6 +220,15 @@ Required | &nbsp;
  `environmentVariableValue` <br/>*string* | An environment variable's value.
  `secretEnvironmentVariableKey` <br/>*string* | The secret environment variable's key.
  `secretEnvironmentVariableValue` <br/>*string* | A sensitive environment variable that should not be exposed.
+ `deploymentInstancePerPops`<br/>*integer* | The number of deployments per point of presence. Only required if autoscaling is not enabled.
+ `cpuUtilization` <br/>*int* | Specify the percentage of CPU utilization.
+ `minInstancesPerPop` <br/>*int* | The minimum number of instances per PoP.
+ `maxInstancesPerPop` <br/>*int* | The maximum number of instances per PoP.
+ `publicPort`<br/>*string* | A single port, such as 80 or a port range, such as 1024-65535 for which a network policy rule will be created for the workload.
+ `publicPortSrc`<br/>*string* | A subnet that will define all the IPs allowed by the network policy rule.
+ `publicPortDesc`<br/>*string* | A summary of what the network policy rule does or a name for it. It is highly recommended to give a unique description to easily identify a network policy rule.
+ `protocol`<br/>*string* | Protocol for the network policy rule. Supported protocols are: `TCP`, `UDP` and `TCP_UDP`.
```

Comment:
I tried to create a workload without specifying the protocol and it is a required field. Should remove this field from the `Optional` and put it in the `Required`

Reply:
The issue is that if even one of these 4 fields is present then all of them are expected but these fields are optional in the sense that if we won't pass all 4 even then the workload will be created.
Due to this, it is unclear where to put these 4 fields.

Reply:
I gotchu! thks for the clarification!

Reply:
I will see if I can add some sort of info section in API docs to mention that.

Reply:
2aaca9e